test/chisel2.py

```diff
@@ -129,12 +130,8 @@ def auth_and_issue(domains, chall_type="http-01", email=None, cert_output=None,
         raise Exception("invalid challenge type %s" % chall_type)
 
     try:
-        while True:
-            order, response = client.poll_order(order)
-            print order.to_json()
-            if order.body.status != "pending":
-                break
-            time.sleep(1)
+        order = client.poll_order_and_request_issuance(order)
+        print(order.fullchain_pem)
```

Comment:
Is this `print` necessary? The server's response to the certificate path GET is already being echoed to stdout and so I think the PEM ends up being printed twice. Maybe it would be better to parse the PEM & print the certificate details textually?